app/logic/bonner.py

```diff
@@ -61,7 +61,10 @@ def getBonnerCohorts(limit=None, currentYear=date.today().year):
     defaultStart = currentYear-4
     firstYear = years[0].year if len(years) and years[0].year < defaultStart else defaultStart
 
-    cohorts = { year: [] for year in range(firstYear, currentYear+1) }
+    defaultEnd = currentYear
```

Comment:
I like that this variable name is descriptive but I think it may be a little unnecessary. Couldn't we just use currentYear wherever we need to use defaultEnd?

Reply:
This is a good suggestion, I refactored the way we were calculating these in their entirety to remove both default variables since we were essentially performing a conditional `min()` and `max()` operation on these.